docs/_docs/snapshots/snapshots.adoc

```diff
@@ -146,29 +146,23 @@ See the link:tools/control-script#checking-snapshot-consistency[Control Script]
 commands.
 
 == Restoring From Snapshot
-
-Currently, the data restore procedure has to be performed manually. In a nutshell, you need to stop the cluster,
-replace persistence data and other files with the data from the snapshot, and restart the nodes.
+=== Manual snapshot restore procedure
+In a nutshell, you need to stop the cluster, replace persistence data and other files with the data from the snapshot, and restart the nodes.
```

Comment:
In a nutshell carries no meaning, and is not needed in docs.

"Stop the cluster, then replace persistence data and other files with the data from the snapshot, and restart the nodes."

Reply:
In this case, a short description is given first, followed by a more detailed description of the same steps.
It seems to be confusing if you do not specify "in a nutshell", isn't it?

Reply:
You do not really need connective tissue like that in docs. The less someone needs to read the better.

In a nutshell also breaks the tone of narrative by making it more conversational, and in the new version it is the first sentence after the header.

Reply:
Removed "In a nutshell", thanks!